.vib/schema-registry/cypress/cypress/integration/schema_registry_spec.js

```diff
@@ -0,0 +1,35 @@
+/// <reference types="cypress" />
+import body from '../fixtures/schema.json';
```

Comment:
Minor style comment:

```suggestion
import { body } from '../fixtures/schema.json';
```

Reply:
Done 👍